examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/QuickStartWithSyncTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.mongodb.realm.realmkmmapp
+
+import io.realm.kotlin.Realm
+import io.realm.kotlin.ext.query
+import io.realm.kotlin.internal.platform.runBlocking
+import io.realm.kotlin.mongodb.App
+import io.realm.kotlin.mongodb.Credentials
+import io.realm.kotlin.mongodb.subscriptions
+import io.realm.kotlin.mongodb.sync.SyncConfiguration
+import io.realm.kotlin.query.RealmResults
+import io.realm.kotlin.types.ObjectId
+import io.realm.kotlin.types.RealmObject
+import io.realm.kotlin.types.annotations.PrimaryKey
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.time.Duration
+
+class QuickStartWithSyncTest: RealmTest() {
+
+    class Task: RealmObject {
+        @PrimaryKey
+        var _id: ObjectId = ObjectId.create()
+        var name: String = ""
+        var status: String = "Open"
+    }
+
+    @Test
+    fun QuickStartWithSyncTest() {
+        val YOUR_APP_ID = FLEXIBLE_APP_ID
+        println("START-----------------")
+
+
+        // :snippet-start: qs-with-sync-open-a-realm
+        val app = App.create(YOUR_APP_ID)
+        val NAME_QUERY = "NAME_QUERY"
+        runBlocking{
```

Comment:
```suggestion
        runBlocking {
```
You might want to start using a linter to keep your syntax consistent. I believe Android Studio has a few options built-in nowadays.

Reply:
Nate! Thanks for the tip, found out you can run gradlew lint and also command+option+L will format any highlighted text, pretty neat that we don't have to add any additional configuration or import 3rd party packages to lint and format